app/src/main/java/com/github/sdpteam15/polyevents/model/database/remote/matcher/Query.kt

```diff
@@ -0,0 +1,110 @@
+package com.github.sdpteam15.polyevents.model.database.remote.matcher
+
+import androidx.annotation.NonNull
+import com.google.android.gms.tasks.Task
+import java.util.*
+
+/**
+ * A Query from a database
+ */
+interface Query {
```

Comment:
What's the difference between this version and Firestore's? Is this in case we decide to change databases?

Reply:
Yes, this in case we decide to change databases